backend/Core/IRepositories/IOrganisationRepository.cs

```diff
@@ -0,0 +1,8 @@
+using Core.DomainModels;
+
+namespace Core.IRepositories;
```

Comment:
Namespacet her kan være Repositories, eller så kan du vurdere vertikale slices, så du samler ikke repositories basert på at de er repositories, men heller legger dem i samme mappe som tingene de agerer på. Altså at alle Project models og repos ligger sammen, osv. for Organizations osv.

Reply:
Vertikale slices er fint! Ordna det